src/server/api/user.js

```diff
@@ -174,10 +174,22 @@ export const resolvers = {
   },
   User: {
     ...mapFieldsToModel(
-      ["id", "firstName", "lastName", "alias", "email", "cell", "assignedCell", "terms"],
+      [
```

Comment:
This looks like editor/lint churn.  I guess leave it, but please debug your settings -- this doesn't look like there are any changes right?